test/lint/lint-all.sh

```diff
@@ -18,11 +18,23 @@ LINTALL=$(basename "${BASH_SOURCE[0]}")
 
 EXIT_CODE=0
 
+IGNORE_EXIT=(
+  "lint-assertions.sh"
+  "lint-circular-dependencies.sh"
+  "lint-format-strings.sh"
+  "lint-includes.sh"
+  "lint-python-dead-code.sh"
+  "lint-python.sh"
+  "lint-tests.sh"
```

Comment:
```suggestion
  "lint-python-dead-code.sh"
  "lint-python.sh"
```
These were fixed by #2476.